uchecker.py

```diff
@@ -68,9 +70,27 @@ def linux_distribution():
     return distro_id + distro_version_id
 
 
+def get_patched_data():
+    result = {}
+
+    if os.path.isfile(LIBCARE_CTL):
+        try:
+            output = subprocess.check_output([LIBCARE_CTL, 'info', '-j'])
+            for line in output.splitlines():
+                item = json.loads(line)
+                for k, v in item.items():
+                    if isinstance(v, dict) and 'buildid' in v:
+                        result[(item['pid'], v['buildid'])] = k
```

Comment:
@grubberr why do we need a `k`? What if we settle with {pid:[buid-id,..]} or just (pid, build-id) ?